src/ResourceTile/ResourceTile.less

```diff
@@ -184,7 +184,3 @@ button.ResourceTile--actionButton {
 .ResourceTile--size--small .ResourceTile--actionButton {
   font-size: @fontSizeMedium;
 }
-
-.DraggableWrapper--dragging .ResourceTile--draggableIcon {
```

Comment:
Since DraggableWrapper is a Launchpad specific component that isn't yet in Dewey, I think it makes sense for this style to live in Launchpad instead of here.